fonduer/supervision/lf_helpers.py

```diff
@@ -1,125 +1,9 @@
-    if span0.get_word_start() < span1.get_word_start():
-        for ngram in get_right_ngrams(
-            span0,
-            window=distance - 1,
-            attrib=attrib,
-            n_min=n_min,
-            n_max=n_max,
-            lower=lower,
-        ):
-            yield ngram
-    else:  # span0.get_word_start() > span1.get_word_start()
-        for ngram in get_left_ngrams(
-            span1,
-            window=distance - 1,
-            attrib=attrib,
-            n_min=n_min,
-            n_max=n_max,
-            lower=lower,
-        ):
-            yield ngram
-
-
-def get_left_ngrams(span, window=3, attrib="words", n_min=1, n_max=1, lower=True):
-    """Get the ngrams within a window to the *left* of the Candidate from its sentence Context.
-
-    For higher-arity Candidates, defaults to the *first* argument.
-
-    :param span: The Span to evaluate. If a candidate is given, default to its first Span.
-    :param window: The number of tokens to the left of the first argument to return
-    :param attrib: The token attribute type (e.g. words, lemmas, poses)
-    :param n_min: The minimum n of the ngrams that should be returned
-    :param n_max: The maximum n of the ngrams that should be returned
-    :param lower: If True, all ngrams will be returned in lower case
-    :rtype: a *generator* of ngrams
-    """
-    span = span if isinstance(span, TemporarySpan) else span[0]  # get first Span
-    i = span.get_word_start()
-    for ngram in tokens_to_ngrams(
-        getattr(span.sentence, attrib)[max(0, i - window) : i],
-        n_min=n_min,
-        n_max=n_max,
-        lower=lower,
-    ):
-        yield ngram
-
-
-def get_right_ngrams(span, window=3, attrib="words", n_min=1, n_max=1, lower=True):
-    """Get the ngrams within a window to the *right* of the Candidate from its sentence Context.
-
-    For higher-arity Candidates, defaults to the *last* argument.
-
-    :param span: The Span to evaluate. If a candidate is given, default to its last Span.
-    :param window: The number of tokens to the left of the first argument to return
-    :param attrib: The token attribute type (e.g. words, lemmas, poses)
-    :param n_min: The minimum n of the ngrams that should be returned
-    :param n_max: The maximum n of the ngrams that should be returned
-    :param lower: If True, all ngrams will be returned in lower case
-    :rtype: a *generator* of ngrams
-    """
-    span = span if isinstance(span, TemporarySpan) else span[-1]  # get last Span
-    i = span.get_word_end()
-    for ngram in tokens_to_ngrams(
-        getattr(span.sentence, attrib)[i + 1 : i + 1 + window],
-        n_min=n_min,
-        n_max=n_max,
-        lower=lower,
-    ):
-        yield ngram
+from fonduer.supervision.utils.structural import *
+from fonduer.supervision.utils.tabular import *
+from fonduer.supervision.utils.textual import *
+from fonduer.supervision.utils.visual import *
```

Comment:
I think the correct way to organize this so that the imports to the user is the same is to make lf_helpers the directory with an __init__.py, and having the separate modaility helper files in that directory, rather than making a utils directory. If that sounds OK to you, I can take care of that update.

Reply:
That's sound right! Thanks!